rails/app/models/development.rb

```diff
@@ -93,7 +93,7 @@ def self.to_shp
     arguments << database['database']
     arguments << %("#{sql}") # %Q["SELECT * FROM developments;"]
 
-    `PGOPTIONS="-c search_path=#{database['schema_search_path']}" pgsql2shp #{arguments.join(" ")}`
+    puts `PGOPTIONS="-c search_path=#{database['schema_search_path']}" pgsql2shp #{arguments.join(" ")}`
```

Comment:
I thought puts is like print. Does it actually run the part after puts here?

Reply:
Ah right, this is a good example of how cryptic ruby can be sometimes. Using backticks  (\`example command\`)  for invoking shell commands in ruby returns stdout in the form of a string. So this line calls pgsql2shp with the proper arguments, then captures the output, returning a string, which we print with puts. 

Backticks look a lot like single quoted strings so this could be very confusing. There are more foolproof ways to perform this same action that we could look into if you'd like. I was looking at [open3](https://docs.ruby-lang.org/en/2.4.0/Open3.html).